### PR TITLE
Pm 2019 04 container

### LINKED ISSUE #2016: running container isn't shown as running (can't see when filter running is set)

### PR DESCRIPTION
**Pull request by phillimorland**
fixes #2016
Also fixes colour issue on ended banner, was not in issues but was discussed in meeting with Sandy and Caro

### CODE REVIEW

File: meinberlin/assets/scss/components/_status_bar.scss [hunk 1]
```diff
@@ -33,8 +33,8 @@
 
 .status-bar__past {
     padding: $padding;
-    background-color: $brand-secondary;
-    color: contrast-color($brand-secondary);
+    background-color: $blue-light;
+    color: $print-black;
```
Comment (fuzzylogic2000) [inline meinberlin/assets/scss/components/_status_bar.scss]: This is a colour, we use for the printing media-query. Isn't there another black we use everywhere else? Like $text-colour? Alright, I checked that. So, we are sure, that this should use another bg and another text-colour than everywhere else on the site?

Comment (fuzzylogic2000) [reply meinberlin/assets/scss/components/_status_bar.scss]: Just say yes, we are very sure and I will merge! :)

**Comment by phillimorland** [comment]
@fuzzylogic2000 it now fails tests?! is this because the test coverage is low for the site?

File: meinberlin/assets/scss/components/_status_bar.scss (hunk 1 above)
Comment (fuzzylogic2000) [reply meinberlin/assets/scss/components/_status_bar.scss]: Cool! Which header? The berlin-one? It should be in variables already then! Maybe bg-secondary or bg-tertiary?

File: meinberlin/assets/scss/components/_portal_header.scss [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 .portal-header {
     @include clearfix;
-    background-color: #bfd3ff;
+    background-color: $blue-light;
```
Comment (fuzzylogic2000) [inline meinberlin/assets/scss/components/_portal_header.scss]: OMG! There was a colour that wasn't a variable! Yeah for fixing that!

**Pull request merged**